pkgs/tools/admin/pgadmin/default.nix

```diff
@@ -72,16 +72,9 @@ let
     azure-identity
   ];
 
-  # override necessary on pgadmin4 6.12
-  pythonPackages = python3.pkgs.overrideScope (final: prev: rec {
-    werkzeug = prev.werkzeug.overridePythonAttrs (oldAttrs: rec {
-      version = "2.0.3";
-      src = oldAttrs.src.override {
-        inherit version;
-        sha256 = "sha256-uGP4/wV8UiFktgZ8niiwQRYbS+W6TQ2s7qpQoWOCLTw=";
-      };
-    });
-  });
+  # keep the scope, as it is used throughout the derivation and tests
+  # this also makes potential future overrides easier
+  pythonPackages = python3.pkgs;
 
 in
 
```

Comment:
sphinx-build should come from nativeBuildInputs.

Reply:
I am not sure what you are referring to. `sphinx` is only used to build the documentation in `preBuild`. It is neither in `nativeBuildInputs`, nor in `buildInputs`, nor in `propagatedBuildInputs`. What do you mean?

Reply:
If you just interpolate a package directly into builder sh like that, the package will come from `pkgsHostTarget` (the package set corresponding to `buildInputs`).  This is incorrect if the package is to be run on the build machine during the build, so **you should never do direct interpolation for build dependencies without explicitly going through the proper package set (e.g. `pkgsBuildHost`)**.  So instead you should add sphinx to `nativeBuildInputs`, and run it from `PATH` during `preBuild`.

Reply:
@alyssais awesome, thanks for the detailed explanation! 
I updated the PR accordingly. :+1: